### PR TITLE
Cleanup trailing whitespace

### PR DESCRIPTION
**Pull request by tedmiston**
### Description

This PR removes trailing whitespace from all files repo-wide.

### Checklist
- [x] Code compiles correctly (i.e `make build`)
- [x] Added tests that cover your change (if possible)
- [x] All tests passing (i.e. `make test`)
- [x] Added/modified documentation as required (such as the README)
- [x] Added yourself to the `humans.txt` file

Note: Added self to humans.txt in #211.

### CODE REVIEW

**Comment by tedmiston** [comment]
The CI failure looks unrelated.  Closing and reopening to re-run.

**Comment by tedmiston** [comment]
Alright that didn't work — can someone with access on Circle CI trigger a re-run on this build?

**Comment by dholbach** [comment]
I re-ran the build, it still fails.

**Comment by richardcase** [comment]
It looks like the 'make test' target is failing. I think perhaps these parts

https://github.com/weaveworks/eksctl/blob/master/Makefile#L17

I can take a look if you want?

**Comment by dholbach** [comment]
Thanks Richard. I just figured out the same thing over here. :-D
Maybe best to back out the changes in `pkg/nodebootstrap/assets.go` for now?

**Comment by richardcase** [comment]
> Maybe best to back out the changes in pkg/nodebootstrap/assets.go for now?

That should solve it. I was just going to try one other thing quickly......

**Comment by richardcase** [comment]
@dholbach @tedmiston - to fix this i think the best option is to run `make generate` and commit the change to [assets.go](https://github.com/weaveworks/eksctl/blob/master/pkg/nodebootstrap/assets.go). As there is a change to [kubeconfig.yaml](https://github.com/weaveworks/eksctl/blob/master/pkg/nodebootstrap/assets/kubeconfig.yaml) it means the assets file is changed which stops the test running as the change isn't committed.

File: pkg/nodebootstrap/assets/kubeconfig.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ contexts:
 - name: kubelet
   context:
     cluster: kubernetes
-    user: kubelet  
+    user: kubelet
```
Comment (richardcase) [inline pkg/nodebootstrap/assets/kubeconfig.yaml]: As there is a change here please run `make generate` and commit the change to assets.go

**Comment by richardcase** [comment]
@dholbach @tedmiston - Do you think its worth us adding a .editorconfig as a follow up? I think it would be good.

**Comment by dholbach** [comment]
Yes, `.editorconfig` might be a good idea to do. :-)

File: pkg/nodebootstrap/assets/kubeconfig.yaml (hunk 1 above)
Comment (errordeveloper) [reply pkg/nodebootstrap/assets/kubeconfig.yaml]: This file has been delete in #184

Comment (richardcase) [reply pkg/nodebootstrap/assets/kubeconfig.yaml]: Great, so rebasing on master should make the build pass then.

Comment (richardcase) [reply pkg/nodebootstrap/assets/kubeconfig.yaml]: Good spot @errordeveloper

**Comment by errordeveloper** [comment]
I've cherry-picked this onto master manually (bc1ab3b0d34d2ae0d9974f73e7a7c119de37d5cb). Thanks!

**Pull request closed without merging**